### PR TITLE
Tier changes, part 2/3

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

This PR implements a number of changes outlined in my Tier audit: https://docs.google.com/document/d/1Go2vRf40PlVWka41dG_jmNZY_xkhhBbclf28duWwZ1E/edit?usp=sharing

Specifically, this PR updates any resources that use an affected image in an example to using one that will remain avaiable after the change. It also fixes the vulnerability comparisions template to point readers to the image's readme in the Image's Directory.

Part 1/3 is Erika's initial work to improve/clarify language around our pricing tiers
Part 3/3 will be implementing a short code note and adding it to resources that will not be updated and instead continue to use images.

### What should this PR do?

Resolves https://github.com/chainguard-dev/internal/issues/4126

### Why are we making this change?

Clarity around products and pricing

### What are the acceptance criteria? 

Changes should make sense. They're generally very small / trivial to the bulk of the respective guide.

### How should this PR be tested?

Give changes a close look please!

### CODE REVIEW

File: content/chainguard/chainguard-registry/pull-through-guides/artifact-registry-pull-through/index.md [hunk 1]
```diff
@@ -59,13 +59,13 @@ gcloud auth configure-docker us-central1-docker.pkg.dev
 Be sure to change `us-central1` to reflect the location of your Artifact Registry repository. 
 Also, after running this command you may be prompted to log in to your Google Cloud account.
 
-After running the command, you will be able to pull a Chainguard Developer Image through Google Artifact Registry. The following example pulls the `wolfi-base` Image:
```
Comment (amouat) [inline content/chainguard/chainguard-registry/pull-through-guides/artifact-registry-pull-through/index.md:62]: I think wolfi-base is ok, but no problem changing

File: content/chainguard/chainguard-registry/pull-through-guides/artifactory-pull-through/index.md [hunk 2]
```diff
@@ -137,13 +137,13 @@ Be sure to include your own username and Artifactory instance.
 
 After running this command, you'll be prompted to enter a password. Copy the token from the second code block, paste it into your terminal, and press `ENTER`.
 
-After running the `docker login` command, you will be able to pull a Chainguard Production Image through Artifactory. The following example pulls the `wolfi-base` Image.
+After running the `docker login` command, you will be able to pull a Chainguard Production Image through Artifactory. The following example will pull the `argo-cli-fips` Image if your organization has access to it:
```
Comment (amouat) [inline content/chainguard/chainguard-registry/pull-through-guides/artifactory-pull-through/index.md:140]: I wonder if chainguard-base would be a better choice (more common and clearer what it is), but it doesn't make much difference.

**Review by amouat — APPROVED** [review]
LGTM. Very minor comments.

File: content/chainguard/chainguard-registry/pull-through-guides/artifactory-pull-through/index.md (hunk 2 above)
Comment (SharpRake) [reply content/chainguard/chainguard-registry/pull-through-guides/artifactory-pull-through/index.md:140]: I like that idea, will change this

**Pull request merged**